### PR TITLE
[feature] sub-packages

### PR DESCRIPTION
**Pull request by LucasBassetti**
#### Introduction

This PR adds the option to transform models using sub-packages as prefixes (#14).

#### How to use

Add `prefixPackages: true` in the transformation options

#### Example 

```
@prefix : <https://example.com#>.
@prefix processModel: <https://example.com/processModel#>.
@prefix ontology: <https://example.com/ontology#>.
@prefix core: <https://example.com/core#>.
@prefix wlzLongTermCare: <https://example.com/wlzLongTermCare#>.
@prefix package: <https://example.com/package#>.
@prefix 1Request: <https://example.com/1Request#>.
@prefix 2Indication: <https://example.com/2Indication#>.
@prefix 3Allocation: <https://example.com/3Allocation#>.
@prefix 4ServiceAgreement: <https://example.com/4ServiceAgreement#>.
@prefix 6TransferMoving: <https://example.com/6TransferMoving#>.
@prefix 0WlzRight: <https://example.com/0WlzRight#>.
@prefix preference: <https://example.com/preference#>.
@prefix 0LthClient: <https://example.com/0LthClient#>.
@prefix 5ServiceDelivery: <https://example.com/5ServiceDelivery#>.
@prefix 7HealthcareServicePackage: <https://example.com/7HealthcareServicePackage#>.
```

#### Extra

I changed all files generated locally to a file name `examples.test.ts`. This will make easier track all the files we want to test.

### CODE REVIEW

File: test_models/valids/index.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const istandard = require('./istandard.json');
 const person = require('./person.json');
 const partWhole = require('./partWhole.json');
 const annotations = require('./annotations.json');
+const packages = require('./packages.json');
```
Comment (tgoprince) [inline test_models/valids/index.ts]: I added this new test model to isolate the tests related to multi-package transformation.

Comment (LucasBassetti) [reply test_models/valids/index.ts]: Root package name fixed!

**Comment by tgoprince** [comment]
I found a weird behavior in the transformation of the istandard model.
I think it is related to the transformation of `<<derivation>>` relations.

**Comment by LucasBassetti** [comment]
@tgoprince fixed!

**Pull request merged**